packages/ui-wallets/src/WalletModal.tsx

```diff
@@ -416,11 +416,11 @@ export function WalletModalV2<T = unknown>(props: WalletModalV2Props<T>) {
         .then((v) => {
           if (v) {
             localStorage.setItem(walletLocalStorageKey, wallet.title)
-          }
-          try {
-            onWalletConnectCallBack?.()
-          } catch (e) {
-            console.error(e)
+            try {
+              onWalletConnectCallBack?.(wallet.title)
+            } catch (e) {
+              console.error(e)
```

Comment:
Can u add the `wallet.title` here easy for us to track the log.

Reply:
Yup. Wdty?
console.error(`${wallet.title},  ${e}`)

Reply:
sure

Reply:
Done